frame/contracts/src/benchmarking/mod.rs

```diff
@@ -1584,16 +1585,26 @@ benchmarks! {
 		let callee_bytes = callees.iter().flat_map(|x| x.account_id.encode()).collect();
 		let value: BalanceOf<T> = 0u32.into();
 		let value_bytes = value.encode();
-		let value_len = value_bytes.len();
+		let value_len = BalanceOf::<T>::max_encoded_len() as u32;
+		// Set an own limit every 2nd call
+		let own_limit = (u32::MAX - 100).into();
```

Comment:
Why is this `own_limit` set on every 2nd call? Shouldn't we account for the worst case scenario only? Not sure though what would be the worst case scenario, the own limit or the parent limit.

Reply:
> Not sure though what would be the worst case scenario, the own limit or the parent limit.

Exactly. That's why it was decided to have them both 50/50